Add CORS details (#139)
decentralized-identity/trustdidweb: spec/security_and_privacy.md

- [UTS-46](https://unicode.org/reports/tr46/)
- [[spec:RFC5895]]

### Cross-Origin Resource Sharing (CORS) Policy Considerations

> To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] should be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response can be set to include the following header:

Author: ```suggestion
To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] needs to be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response MUST be set to include the following header:
```

> 

Reviewer: “can” — should that be a “SHOULD” in spec language, or in this part of the spec is that not appropriate?

Author: good call - maybe even a MUST?

> To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] needs to be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response MUST be set to include the following header:

Reviewer 2: ```suggestion
To support scenarios where DID resolution is performed by client applications running in a web browser, the file served for the [[ref: DID Log]] needs to be accessible by any origin. To enable this, the [[ref: DID Log]] HTTP response MUST include the following header:
```

A bit less wordy
